scripts/update-venue-owners.ts

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import {
+  checkFileExists,
+  FieldValue,
+  findUserByEmail,
+  initFirebaseAdminApp,
+  makeScriptUsage,
+  parseCredentialFile,
+} from "./lib/helpers";
+
+// ---------------------------------------------------------
+// Configuration (this is the bit you should edit)
+// ---------------------------------------------------------
+
+const VENUES_TO_CHANGE_OWNERSHIP: string[] = [];
+
+const EMAIL_ADDRESSES: string[] = [];
+
+// ---------------------------------------------------------
+// HERE THERE BE DRAGONS (edit below here at your own risk)
+// ---------------------------------------------------------
+
+enum Mode {
+  Add = "ADD",
+  Remove = "REMOVE",
+}
+
+const VALID_MODES: string[] = [Mode.Add, Mode.Remove];
+
+const usage = makeScriptUsage({
+  description: `Bulk add users to the owners list based on the supplied email address(es).\n\n(Note: the accounts must already exist.)`,
+  usageParams: `CREDENTIAL_PATH [${Mode.Add}/${Mode.Remove}] EMAIL1 EMAIL2 EMAIlN...`,
```

Comment:
I would add the venueIds here as well, using the venueId1,venueId2,etc pattern used in other scripts, probably before the ADD

Reply:
a good idea. 🙃 
added: https://github.com/sparkletown/sparkle/pull/1520/commits/c9c2aefa61c3f70c9702ee5722905f14a693a29e

Reply:
a good idea. 🙃 
added: https://github.com/sparkletown/sparkle/pull/1520/commits/c9c2aefa61c3f70c9702ee5722905f14a693a29e

Reply:
I meant where they are all literally touching with a comma between them, then using split to get the array. That’s how the existing scripts do it.

I’m concerned about searching for position/etc in the current way in case there’s a name clash/etc that leads to weird behaviour